app/code/Magento/Translation/Block/Js.php

```diff
@@ -68,4 +68,12 @@ public function getTranslationFilePath()
     {
         return $this->fileManager->getTranslationFilePath();
     }
+
+    /**
+     * @return string
+     */
+    public function getTranslationFileFullPath()
+    {
+        return $this->fileManager->getTranslationFileFullPath();
```

Comment:
> I'm not sure if this breaks the backwards compatibility rules, since it adds a public method to an `@api` annotated class?

This change indeed breaks backwards compatibility rules, but with existing block design this  case becomes quite complicated to solve properly without adding a method.

If we are going to violate the rule for the specific case – I would suggest implementing this in the "desirable state" way, so it would cover all the theoretical cases which may come up in the future.

Briefly, this could be a method with more general name, something like `getTranslationFileVersion`, which would proxy the call to the dedicated model, which calculates the value (doing so in the template is not a best practice anyway). This way it becomes a more usable "extension point" for third-party developers to possibly customize the behavior and improves ability to maintain this code for the core team.